src/index.html

```diff
@@ -5,8 +5,310 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
   </head>
+  <header class="header">
+    <div class="header__logo">
+      <a href="#home">
+        <img src="images/logo.png" alt="logo.png">
+      </a>
+    </div>
+    <nav class="nav">
+      <ul class="nav__list">
+        <li class="nav__item">
+          <a class="nav__link nav__link--active is-active" href="#apple">apple</a>
+        </li>
+        <li class="nav__item">
+          <a class="nav__link" href="#samsung">samsung</a>
+        </li>
+        <li class="nav__item">
+          <a class="nav__link" href="#phone">смартфоны и телефоны</a>
+        </li>
+        <li class="nav__item">
+          <a class="nav__link"
+             data-qa ="nav-hover"
+             href="#comp">ноутбуки и компьютеры
+          </a>
```

Comment:
Ты правильно сделал, что начал переносить атрибуты, но перенос атрибутов тега надо начинать с первого атрибута включительно, а не оставлять его в одной строке с открывающим тегом. При этом закрывающая стрелка открывающего тега так же выносится на следующую строку, и по уровню выравнивается с началом открывающей стрелки. Контент в таком случае также желательно вставлять между отдельной строкой между тегами, с отступом вложенности.
```suggestion
          <a
            class="nav__link"
            data-qa ="nav-hover"
            href="#comp"
          >
            ноутбуки и компьютеры
          </a>
```